zipline/gens/tradesimulation.py

```diff
@@ -256,13 +299,13 @@ def _process_snapshot(self, dt, snapshot, instant_fill):
             # Now that handle_data has been called and orders have been placed,
             # process the event stream to fill user orders based on the events
             # from this snapshot.
-            for event in events_to_be_processed:
-                for txn, order in blotter_process_trade(event):
+            for trade in events_to_be_processed:
+                for txn, order in blotter_process_trade(trade):
```

Comment:
It makes sense to me to have both:

1. If instant_fill, the blotter processes all trades on the same cycle after handle_data.
2. If not instant_fill, the blotter processes all trades on the same cycle before handle_data.

Right?